host/entry_android.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2018 The ZikiChomgo Authors. All rights reserved.  Use of this source
+// code is governed by a license that can be found in the License file.
+
+package host
+
+// Names names the sound system entry points for the host.
+func Names() []string {
+	return [...]string{"AudioFlinger", "TinyALSA", "ALSA"}
```

Comment:
Does this compile? It seems an array is returned in place of a slice?

Could use:
```go
[...]string{"AudioFlinger", "TinyALSA", "ALSA"}[:]
```

Reply:
just simplified it so that each entry_{host}.go contains just the array, and the function compiles, will push it in a minute.